scripts/rollup/build.js

```diff
@@ -394,6 +447,7 @@ function getPlugins(
     ),
     // Remove 'use strict' from individual source files.
     {
+      name: "remove 'use strict'",
       transform(source) {
         return source.replace(/['"]use strict["']/g, '');
       },
```

Comment:
Yeah, I did see that.  However, it's not relevant in this case, because we _don't_ care about preserving sourcemaps from any of the steps before the `closure` plugin, and we _don't_ care about sourcemaps for any of the individual source files.  We just need the Closure bundle minification step sourcemap as the baseline.